torchvision/transforms/v2/_color.py

```diff
@@ -199,10 +224,10 @@ class RandomPhotometricDistort(Transform):
 
     def __init__(
         self,
+        brightness: Tuple[float, float] = (0.875, 1.125),
```

Comment:
This is just to align the signature with the logic below, right?

Reply:
yes sorry I forgot to comment - this is to align with the signature of ColorJitter